emission/storage/decorations/timeline.py

```diff
@@ -49,6 +49,52 @@ def get_timeline(user_id, start_ts, end_ts):
 
     return Timeline(places, trips)
 
+
+
+def get_aggregate_timeline_from_dt_box(start_dt, end_dt, box):
```

Comment:
These two functions seem really similar. Refactor for commonality? It seems like you could have a single function with box=None if you had no box, and then pass it through to `get_aggregate_places`, simplifying the flow overall.